pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v2/PersistentTopics.java

```diff
@@ -149,8 +182,14 @@ public void createPartitionedTopic(@PathParam("tenant") String tenant, @PathPara
         @ApiResponse(code = 412, message = "Failed Reason : Name is invalid or Namespace does not have any clusters configured"),
         @ApiResponse(code = 503, message = "Failed to validate global cluster configuration")
     })
-    public void createNonPartitionedTopic(@PathParam("tenant") String tenant, @PathParam("namespace") String namespace,
+    public void createNonPartitionedTopic(
+            @ApiParam(value = "Specify the tenant to which this topic belongs", required = true)
+            @PathParam("tenant") String tenant,
+            @ApiParam(value = "Specify the namespace to which this topic belongs", required = true)
+            @PathParam("namespace") String namespace,
+            @ApiParam(value = "Specifies topic name", required = true)
```

Comment:
I am not sure if we need to use “Specify" here. or just use "The tenant of the topic"?

Reply:
If it is a field to be filled by user, we can use "specify".